extensions/amp-form/0.1/amp-form.js

```diff
@@ -721,7 +721,9 @@ function isDisabled_(element) {
 function installSubmissionHandlers(win) {
   onDocumentReady(win.document, doc => {
     toArray(doc.forms).forEach((form, index) => {
-      new AmpForm(form, `amp-form-${index}`);
+      if (!form.classList.contains('-amp-form')) {
```

Comment:
should i update https://github.com/ampproject/amphtml/blob/master/extensions/amp-form/0.1/amp-form.js#L164 ?